### ISSUE TITLE
Support for regex in Path Param

### ISSUE THREAD

**Issue by rsertelon** [nmorel/gwt-jackson-rest#10]
JAX-RS allows path params to contain regex (for URI matching).

Sadly, there do not seem to be any support for this in gwt jackson rest.

I might be able to try and create a pull request if it helps.

**Comment by nmorel** [comment]
You can try yes :)  
I only give basic support (answering question/merging/deploying) in this project now. 
I'm not using it anymore at work (I don't even use java anymore!)

**Comment by rsertelon** [comment]
Just done it ;-) the PR contains a small change that removes the regex and leaves the library work like usual.

**Comment by rsertelon** [comment]
@nmorel it seems cloudbees discontinued its CI service.

Maybe I can try to setup a travis build?

**Comment by nmorel** [comment]
Thanks for proposing! Travis should work for this project since it does not have unit tests.
But I don't think that's really needed for now just for the release part. 
I need it more for the gwt-jackson project but last time I tried Travis, the GWT tests didn't work.

I managed to retrieve the jenkins job configurations (mostly the big maven command line to release) and can do it manually.

A snapshot should be available with your PR.
